years_to_hours.rb

```diff
@@ -0,0 +1,55 @@
+def run()
+    puts("Welcome! In this app, you'll be converting:
+        \r Years to hours
+        \r Decades to minutes
+        \r Your age to seconds\n")
+    puts(year_hour())
+    puts(decade_minute())
+    puts(age_second())
+    puts("You have reached the end of the program. Bye :)")
+end
+
+def input_handling(what_to_ask)
+    while true do
+        input = gets.chomp
+        # used Regex to check if it's a whole number
+        unless input.match(/[^0-9]/)
+            return input.to_i
+        else
```

Comment:
While this is possible in Ruby, using `unless...else...` makes it harder to read the code. I would suggest sticking to `if...else...` for most cases.